### PR TITLE
[Native Image] Add `native_nightly_packs` To The `conf.json`

### PR DESCRIPTION
**Pull request by shmuel44**
## Related Issues
fixes: [link](https://jira-dc.paloaltonetworks.com/browse/CIAC-11453) to the issue
Validate conf https://github.com/demisto/demisto-sdk/pull/4479

## Description

Add `native_nightly_packs` To The `conf.json`

### CODE REVIEW

File: Tests/conf.json [hunk 1]
```diff
@@ -6124,6 +6124,13 @@
         "Aha": "No instance",
         "SlackV2": "Integration slackV2 is deprecated."
     },
+    "native_nightly_packs":[
```
Comment (dorschw) [inline Tests/conf.json:6127]: make sure to update [here](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/content_graph/objects/conf_json.py) for the `validate-conf-json` hook to pass.
(Will require an SDK release)

1. add this field to the model definition
2. add this field to `linked_content_items`, so we validate the packs here are present in the repo.

Comment (shmuel44) [reply Tests/conf.json:6127]: https://github.com/demisto/demisto-sdk/pull/4479

File: .gitlab/ci/.gitlab-ci.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 variables:
-  CURRENT_BRANCH_NAME: $INFRA_BRANCH
+  CURRENT_BRANCH_NAME: sk_conf_native
```
Comment (shmuel44) [inline .gitlab/ci/.gitlab-ci.yml:2]:
```suggestion
  CURRENT_BRANCH_NAME: $INFRA_BRANCH
```

File: .gitlab/ci/.gitlab-ci.yml [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
 
 include:
   - file: "/.gitlab/ci/content-ci/ci/.gitlab-ci.yml"
-    ref: $INFRA_BRANCH
+    ref: sk_conf_native
```
Comment (shmuel44) [inline .gitlab/ci/.gitlab-ci.yml:6]:
```suggestion
    ref: $INFRA_BRANCH
```

**Pull request merged**